go.mod

```diff
@@ -167,6 +177,7 @@ replace (
 	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.34.2
 	k8s.io/kubectl => k8s.io/kubectl v0.34.2
 	k8s.io/kubelet => k8s.io/kubelet v0.34.2
+	k8s.io/kubernetes => k8s.io/kubernetes v1.34.2
```

Comment:
Do we really need to vendor in k8s.io/kubernetes?  We've had issues with this in the past.  See: https://github.com/openshift/cluster-node-tuning-operator/pull/1356 and https://github.com/openshift/cluster-node-tuning-operator/pull/1332

Reply:
That's fine, let's have a separate require section for 

```
github.com/onsi/ginkgo/v2 => github.com/openshift/onsi-ginkgo/v2 v2.6.1-0.20241205171354-8006f302fd12
```
as mentioned above.  It is a nit, but it shouldn't really be in the `// Pinned to kubernetes-1.34.2` section.  Something like this:

```
// All the pinned dependencies are a technical debt of this or upstream projects that needs to be fixed.
replace (
       github.com/onsi/ginkgo/v2 => github.com/openshift/onsi-ginkgo/v2 v2.6.1-0.20241205171354-8006f302fd12
)
```

Makes sense?